.gitpod.yml

```diff
@@ -0,0 +1,9 @@
+# This configuration file was automatically generated by Gitpod.
```

Comment:
Can you help me understand why this is useful? I'm not familiar with Gitpod

Reply:
oh no I am sorry!

Reply:
OK, so 1 bad thing about gitpod:

It puts these in repos automatically.  

So I guess not seeing them in unrelated PR's could be a good reason to do it, but that's not really a good reason.  Let me get into an actual good reason, after deleting this file.